.pipelines/build.cmd

```diff
@@ -33,9 +33,32 @@ echo Copying contents...
 cd /D "%~dp0"
 if errorlevel 1 goto :error
 
-xcopy /e /i c:\webrtc\src\out ..\output
-if errorlevel 1 goto :error
+call :copy c:\webrtc\src\out\msvc\uwp\Release\x64\obj\webrtc.lib ..\output\msvc\uwp\Release\x64\obj\
+call :copy c:\webrtc\src\api\*.h ..\include\api\
+call :copy c:\webrtc\src\audio\*.h ..\include\audio\
+call :copy c:\webrtc\src\base\*.h ..\include\base\
+call :copy c:\webrtc\src\call\*.h ..\include\call\
+call :copy c:\webrtc\src\common_audio\*.h ..\include\common_audio\
+call :copy c:\webrtc\src\common_video\*.h ..\include\common_video\
+call :copy c:\webrtc\src\data\*.h ..\include\data\
```

Comment:
Please remove this line. The [log file](https://github-private.visualstudio.com/microsoft/_build/results?buildId=7866&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=dcf93f35-c1ba-5dce-0078-8d628250da3a&l=3789) reports that there is no header files in the `data` folder.
```
[INFO] [3720] C:\webrtc\src\common_video\test\utilities.h
[INFO] [3721] 17 File(s) copied
[INFO] [3722] 0 File(s) copied
##[warning][Build] [Build WebRTC] [3723] File not found - *.h
[WARN] [3723] File not found - *.h
[INFO] [3724] C:\webrtc\src\logging\rtc_event_log\fake_rtc_event_log.h
```